WRQ-31515: Fixed `Scroller`, `VirtualList`, and `VirtualGridList` to set prop value to default when `undefined` is passed for the prop value (#3265)
enactjs/enact: packages/ui/CHANGELOG.md

The following is a curated list of changes in the Enact ui module, newest changes on the top.

## [unreleased]

### Fixed

> - `ui/Scroller`, `ui/VirtualList.VirtualList` and `ui/VirtualList.VirtualGridList` to set prop value to default when `undefined` is passed for the prop value

Reviewer: ```suggestion
- `ui/Scroller` and `ui/VirtualList` to have default prop when `undefined` prop is passed
```

Author: It's fixed.
